lib/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicator.scss

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-vulnerability-indicator {
+  align-items: center;
+  box-sizing: content-box;
+  border: 1px solid;
+  border-left-width: 8px;
+  border-radius: $nx-border-radius;
+  display: inline-flex;
+  font-size: $nx-font-size-s;
+  height: 30px;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+  width: max-content;
+}
+
+.nx-vulnerability-indicator__count {
+  @include bold();
+}
+
+.nx-vulnerability-indicator__text {
+  margin-left: $nx-spacing-xs;
+  text-transform: uppercase;
+}
```

Comment:
Should be semi-bold

Reply:
> Should be semi-bold

https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/e876a57eacad29d4171059706217dd98cfec0631